free-2.md

```diff
@@ -11,41 +11,88 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+There is a need to compile and link C/C++ files by the software development community.
+Currently, the common way of compiling and linking C/C++ code is through a DSL called
+make. However, make is very very confusing to use and doesnt seem to have much fluency
+so I think it really needs a revamp. If i could help them, their experience would be
+much less confusing. I imagine they could easily just define the dependencies in a more
+natural way than the Syntactic-Aspartame filled make. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is required for this I feel. Currently lots of C/C++ rely on Make as a DSL
+and I would want my DSL to not be to big a change or too uncomfortable to use meaning
+a DSL is required to do this. I am sure a program could be created instead but it would
+take away from the current measures in place and could even make it more complicated.
+I think a DSL addresses the need since users will need to customize their files on the go
+and maybe even create these make files with scripts which is only possible if it is a DSL
+or has an API like a DSL.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I think my main relation to this is that in CS70 make files were a horrible aspect
+and our groups could never figure them out. They just arent natural and I think my anger
```

Comment:
Personally, I think the biggest problem with make files (and building in general) in CS70 is that they're very much a 2nd, or even 3rd class citizen. They are, at best, taught as an aside in homework instructions, instead of time spent actually explaining how they work. I agree that make is hard to use, and that C++ has a lot of complexity with how you build executables, but I'm not convinced that a new make-based DSL is a better solution.